src/main.js

```diff
@@ -1,84 +1,101 @@
 /* global $ _ Prism Handlebars */
 
-const ROOT_URL = 'https://transportapi.github.io/usage-examples/src/'
+const urlParams = new URLSearchParams(window.location.search)
```

Comment:
Huh, I lot of missing `;` in this file. I think the best practices are still to use `;` so I suggest we fix the style.